frontend/pages/admin/IntegrationsPage/cards/Integrations/_styles.scss

```diff
@@ -83,8 +83,6 @@
     border-radius: $pad-xsmall;
     border: px-to-rem(1) solid $ui-fleet-black-10;
     margin: 0;
-    max-width: none;
```

Comment:
So this also fixes table widths on the settings -> integrations page? Checking as that wasn't mentioned in the PR and if we touch a thing we should QA it.

FWIW EmptyTable is used a lot of places but given that it's a component testing one place should be low-risk enough.

Reply:
Sorry, mentioned in the original ticket under screenshot and reproduce.
There is no design for this fix, I just fixed it so it wouldn't bleed off the screen.
Will add to the description. Ensuring everything is written in 3 places (original bug reproduce steps, QA plan, and PR description feels like relative overkill), maybe because it's 4:30 PM.

https://github.com/fleetdm/fleet/issues/25015

Reply:
Oh yup, sorry, mislabeled as "automation" and fixed to "integration" in my PR description
> - Other part of this ticket: Fix empty integration container from overflowing page down to 768px

Reply:
Sorry, issue desc is good enough. Marking this as resolved.